lib/browser.ts

```diff
@@ -775,8 +775,8 @@ export class ProtractorBrowser extends Webdriver {
         .then(loadMocks, deferred.reject);
 
     let self = this;
-    function loadMocks(angularVersion: number) {
-      if (angularVersion === 1) {
+    function loadMocks(angularVersion: string) {
+      if (angularVersion == '1') {
```

Comment:
`testForAngular` returns `ver` as `number`. [This line](https://github.com/angular/protractor/blob/cc459402970afe5270e54c087a5347592cff9270/lib/browser.ts#L761) is where `string` needs to be replaced with `number`.